pytransform3d/plot_utils/_plot_functions.py

```diff
@@ -131,15 +131,20 @@ def plot_sphere(ax=None, radius=1.0, p=np.zeros(3), ax_s=1, wireframe=True,
         ax = make_3d_axis(ax_s)
 
     phi, theta = np.mgrid[0.0:np.pi:n_steps * 1j, 0.0:2.0 * np.pi:n_steps * 1j]
-    x = p[0] + radius * np.sin(phi) * np.cos(theta)
-    y = p[1] + radius * np.sin(phi) * np.sin(theta)
-    z = p[2] + radius * np.cos(phi)
-
-    if wireframe:
-        ax.plot_wireframe(
-            x, y, z, rstride=10, cstride=10, color=color, alpha=alpha)
-    else:
-        ax.plot_surface(x, y, z, color=color, alpha=alpha, linewidth=0)
+    X = radius * np.sin(phi) * np.cos(theta)
+    Y = radius * np.sin(phi) * np.sin(theta)
+    Z = radius * np.cos(phi)
+    
+    for p in np.array(p, ndmin=2):
```

Comment:
I don't know in which numpy version ndmin was introduced. To be on the safe side, I'd suggest to use `np.atleast_2d`.